app/views/submissions/_result_table.html.erb

```diff
@@ -13,7 +13,7 @@
     <tr>
       <td class="submission-body-name"><%= submission.full_name %></td>
       <td class="submission-mobile-hidden"><%= submission.email %></td>
-      <td class="submission-mobile-hidden"><%= submission.average_rate.round(2) if submission.rated? %></td>
```

Comment:
Because it is already rounded in the `SubmissionPresenter`, I probably forgot to delete it from here. :D